infrastructure/variable.tf

```diff
@@ -198,4 +198,18 @@ locals {
 
   api_gateway_subdomain_name   = "${var.certificate_subdomain_name_prefix}${terraform.workspace}"
   api_gateway_full_domain_name = "${var.certificate_subdomain_name_prefix}${terraform.workspace}.${var.domain}"
+
+  current_region     = data.aws_region.current.name
+  current_account_id = data.aws_caller_identity.current.account_id
+
+  ses_feedback_sender_email_address = (local.is_sandbox_or_test
+    ? "feedback@mailing.ndr-dev.${var.domain}"
+    : "feedback@${module.ndr-feedback-mailbox.mail_from_domain_name}"
+  )
+  feedback_recipient_list_ssm_param_key = (local.is_sandbox_or_test
+    ? "/prs/dev/user-input/feedback-recipient-email-list"
+    : "/prs/${var.environment}/user-input/feedback-recipient-email-list"
+  )
+  ses_send_feedback_email_resource_arn = "arn:aws:ses:${local.current_region}:${local.current_account_id}:identity/${local.ses_feedback_sender_email_address}"
```

Comment:
What is the reason behind using this and not adding an output from the SES module?

Reply:
Just now I edited this a little bit. 
Now this variable will use the output from SES module if enable=true, otherwise will use the current region & account_id to build a workable arn

```terraform
ses_send_feedback_email_resource_arn = (module.ndr-feedback-mailbox.is_enable
    ? module.ndr-feedback-mailbox.ses_domain_identity_arn
    : "arn:aws:ses:${local.current_region}:${local.current_account_id}:identity/*"
  )
```